### PR TITLE
Reduce memory allocations for ConnectUtil

### PR DESCRIPTION
**Pull request by sjoelund**

### CODE REVIEW

File: Compiler/BackEnd/BackendDAEUtil.mo [hunk 1]
```diff
@@ -1317,30 +1317,16 @@ algorithm
...
-      0 = arr[x];
-      res = blockIsDynamic(xs, arr);
-    then res;
-
-    case ((x::_),arr) equation
-      mark_value = arr[x];
-      (mark_value <> 0) = true;
-    then true;
-  end matchcontinue;
+  outBoolean := false;
```
Comment (hkiel) [inline Compiler/BackEnd/BackendDAEUtil.mo]: @sjoelund  this line should come *after* the for loop

Comment (sjoelund) [inline Compiler/BackEnd/BackendDAEUtil.mo]: Copy-paste errors :D

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2016-04-01_15-32-12](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/2165/).

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2016-04-01_16-45-15](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/2166/console).

**Pull request merged**